src/services/dragonchain-client/DragonchainClient.ts

```diff
@@ -665,6 +666,30 @@ export class DragonchainClient {
     throw new FailureByDesign('PARAM_ERROR', 'At least one of `smartContractId` or `transactionType` must be supplied');
   };
 
+  /**
+   * Get a single smart contract by one of id or transaction type
+   */
+  public getSmartContractLogs = async (options: {
+    /**
+     * Contract id to get logs from
+     */
+    smartContractId: string;
+    /**
+     * Tail, the maximum number of logs to return
```

Comment:
Should probably specify in this comment that this should be an integer value only

Reply:
The autodocs don't display type?

Reply:
Well, it displays 'number', but javascript doesn't have an 'integer' type, so it's worth noting in the comment that this can't be a decimal